index.js

```diff
@@ -0,0 +1,126 @@
+var PARAMS = /\{([^\}]*)\}/g;
+var PROPS = /\{\{([^\}]*)\}\}/g;
+
+var crossroads = require('crossroads');
+var objectpath = require('object-path');
+var humanize = require('string-humanize');
+var path = require('path');
+var strip = require('strip');
+var urlRegex = require('url-regex');
+
+function parse( content, context ){
+  if( !content ) return false;
+
+  // Files
+  if( content.split('.').pop().match(/gif|jpg|jpeg|png/) ){
+    return content
+      ? path.join(context.url.origin, content)
+      : false;
+  }
+
+  // URLs
+  if( content && content.slice(0,4) === 'http' ){
```

Comment:
You can remove the first condition here too, because of the first line of the function.